server/job_service.go

```diff
@@ -232,15 +233,20 @@ func (p *Plugin) newJobService(serviceURL string) (*jobService, error) {
 	}, nil
 }
 
-func (s *jobService) RunJob(cfg offloader.JobConfig) (offloader.Job, error) {
+func (s *jobService) RunJob(cfg job.Config) (job.Job, error) {
 	return s.client.CreateJob(cfg)
 }
 
-func (s *jobService) StopJob(jobID string) error {
-	return s.client.StopJob(jobID)
+func (s *jobService) StopJob(channelID string) error {
+	// Since MM-52346, stopping a job really means signaling the bot it's time to leave
+	// the call. We do this implicitly by sending a fake call end event.
+	s.ctx.publishWebSocketEvent(wsEventCallEnd, map[string]interface{}{
+		"channelID": channelID,
+	}, &model.WebsocketBroadcast{UserId: s.ctx.getBotID(), ReliableClusterSend: true})
```

Comment:
Not super happy with this but it's simple enough and a bit more explicit. One alternative could be to don't send anything at all and check on the client (recorder) side for the recording ended state websocket event and disconnect upon receiving that one. Let me know what you think.

Reply:
Either way, doesn't seem to be that big of an issue to me. This is like telling the bot the call is "over", so that's fine. On the other hand, having the recording act on the recording ended websocket event is a bit more logical or "reactive". But up to you.